Cargo.toml

```diff
@@ -40,9 +40,9 @@ console_error_panic_hook = "0.1.7"
 futures-util = "0.3.27"
 http = "0.2.6"
 js-sys = "0.3.49"
-matrix-sdk-common = { git = "https://github.com/matrix-org/matrix-rust-sdk", rev = "c6dab4088e4288a5a0636d8687534aa4652bed18", features = ["js"] }
-matrix-sdk-indexeddb = { git = "https://github.com/matrix-org/matrix-rust-sdk", rev = "c6dab4088e4288a5a0636d8687534aa4652bed18" }
-matrix-sdk-qrcode = { git = "https://github.com/matrix-org/matrix-rust-sdk", rev = "c6dab4088e4288a5a0636d8687534aa4652bed18", optional = true }
+matrix-sdk-common = { git = "https://github.com/matrix-org/matrix-rust-sdk", rev = "2cec3b0c4568a6b8f762ff46d285df710047d2a1", features = ["js"] }
```

Comment:
bumping the rust-sdk version to pick up https://github.com/matrix-org/matrix-rust-sdk/pull/2298